packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -331,6 +345,17 @@ const websiteEntityPage = (
       {errorsContent}
     </EntityLayout.Route>
 
+    <EntityLayout.Route path="/dependencies" title="Dependencies">
+      <Grid container spacing={3} alignItems="stretch">
+        <Grid item md={6}>
+          <EntityDependsOnComponentsCard />
```

Comment:
I notice this also has a `gridItem` variant. I don't know what's best practice here, but I've seen a ton of cards have this variant and it's explicitly defined in the calls from `EntityPage.tsx`.  Wonder if this should be identified here also? (like it is also on line 490 for `EntityHasResourcesCard`.

Reply:
Sure.  Some of the other components, such as the GitHub cards don't have it defined, and it's the default, but I can add it in.  Totally flexible.